VENDORS.md

```diff
@@ -24,67 +25,64 @@ If you notice any mistakes, or would like to contribute, feel free to **make** a
 | [1upKeyboards](https://1upkeyboards.com)                     | [x]      | [x]  | [x]   | [x]    | [x]     | [x]       | New York, NY               |
 | [3DKeebs](https://3dkeebs.com)                               | [x]      | [x]  | [x]   |        |         |           | Dallas, TX                 |
 | [415keys](https://415keys.com)                               | [x]      | [x]  | [x]   |        |         |           | Richmond, NC               |
-| [Alphakeys (US)](https://alphakeys.com/)                     | [x]      |      |       |        |         |           | ?                          |
-| [Asceny](https://asceny.com/)                                |          |      |       | [x]    |         |           | ?                          |
-| [BoardSource](https://boardsource.xyz/)<sup>1</sup>          |          |      |       |        |         | [x]       | ?                          |
+| [Alphakeys (US)](https://alphakeys.com/)                     | [x]      | [x]  | [x]   |        |         |           | ?                          |
+| [Asceny](https://asceny.com/)                                |          |      | [x]   | [x]    |         | [x]       | ?                          |
+| [BoardSource](https://boardsource.xyz/)<sup>1</sup>          | [x]      | [x]  | [x]   |        | [x]     | [x]       | ?                          |
```

Comment:
not sure how long boardsource has been out of lube, do you know

Reply:
no clue

Reply:
I don't see any references to it in their discord since late 2020 but I could've missed something